### ISSUE TITLE
SfBottomSheet not working with screen reader

### ISSUE THREAD

**Issue by Andreas973** [syncfusion/maui-toolkit#310]
### Description

When using the SfBottomSheet control (tested on iOS with VoiceOver), the screen reader focus does not move to the bottom sheet content when it opens. Furthermore, you cannot manually swipe to navigate to the bottom sheet or its elements.

Sample repo: https://github.com/Andreas973/SyncfusionToolkitBottomSheetScreenReader

### Steps to Reproduce

1. Create a new .NET MAUI project.

2. Install the Syncfusion.Maui.Toolkit NuGet package.

3. Add an SfBottomSheet to a page.

4. Add interactive elements inside the Bottom Sheet (e.g., Button, Label, or Entry).

5. Deploy the application to a physical iOS device (or Simulator).

6. Enable VoiceOver in iOS Settings > Accessibility. Or use Accessibility Inspector in XCode.

7. Trigger the bottom sheet to open.

8. Attempt to swipe right/left to navigate to the content inside the sheet.

### .NET Version

.NET 10.0

### Version with bug

1.0.7

### Is this a regression from previous behavior?

Not sure, haven't tested other versions

### Affected platforms

iOS

### Affected Platform Versions

iOS 26

### Relevant log output

```shell

```

**Comment by naveenkumar-sanjeevirayan** [comment]
Hi @Andreas973,

We’ve resolved the issue, and the fix will be included in the upcoming release (v1.0.10).

Regards,
Naveenkumar S
